extensions/ql-vscode/src/data-extensions-editor/yaml.ts

```diff
@@ -116,6 +117,33 @@ export function createDataExtensionYamlsPerLibrary(
   return result;
 }
 
+export function createDataExtensionYamlsForFrameworkMode(
```

Comment:
Something to think about, is that eventually it would be amazing if modelling library X while in application mode for A, would yield the same file as modelling library X in framework mode.

Reply:
Yes, I think this is something to think about. For now, I don't think we can really detect this reliably because we're depending on two different inputs (database vs JAR name).